test/test_artifacts/v2/scripts/run_maxdome_jupyter_server_extension_test.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+pip install pytest-jupyter
+pytest -vv -r ap /opt/conda/lib/python3.10/site-packages/maxdome_jupyter_server_extension || exit $?
```

Comment:
note - the python version might be updated in v2. Would be good to get the install path/python version dynamically, something like https://github.com/aws/sagemaker-distribution/blob/main/template/v1/Dockerfile#L98

Reply:
Fixed with follow up commit.